core/domain/feedback_services.py

```diff
@@ -203,9 +208,29 @@ def create_suggestion(exploration_id, author_id, exploration_version,
         suggestion_content)
 
 
+def _get_suggestion_dict_from_model_instance(suggestion):
+    if suggestion is None:
+        return suggestion
+    return {
+        'id': suggestion.id,
+        'author_id': suggestion.author_id,
+        'exploration_id': suggestion.exploration_id,
+        'exploration_version': suggestion.exploration_version,
+        'state_name': suggestion.state_name,
+        'state_content': suggestion.state_content}
+
+
 def get_suggestion(exploration_id, thread_id):
-    return feedback_models.SuggestionModel.get_by_exploration_and_thread_id(
-        exploration_id, thread_id)
+    return _get_suggestion_dict_from_model_instance(
+        feedback_models.SuggestionModel.get_by_exploration_and_thread_id(
+            exploration_id, thread_id))
+
+
+def _string_to_bool(str):
```

Comment:
This conversion should happen in the controller, not here. Boolean values should be passed to this module. (By the time stuff reaches the domain layer it should be at least minimally valid.)